### PR TITLE
Feature to set bitrates and framebuffersize from configfs

### PR DESCRIPTION
**Pull request by mgrzeschik**
Pulled in the uvc patches from @any1 to write

- dwMaxVideoFrameBufferSize
- dwMinBitRate
- dwMaxBitRate

and fix setting of
- dwDefaultFrameInterval

### CODE REVIEW

File: src/function/uvc.c [hunk 1]
```diff
@@ -99,6 +99,16 @@ struct {
 		.export = usbg_set_config_node_int,		        \
 	}
 
+#define UVC_DEC64_ATTR(_name)						\
+	{								\
+		.name = #_name,						\
+		.offset = offsetof(struct usbg_f_uvc_frame_attrs, _name),     \
+		.get = usbg_get_dec,				        \
+		.set = usbg_set_dec,				        \
```
Comment (mgrzeschik) [inline src/function/uvc.c:107]: @any1 You probably want to use the new implemented usgb_set_dec64 functions here.

File: src/function/uvc.c [hunk 2]
```diff
@@ -99,6 +99,16 @@ struct {
...
+		.import = usbg_get_config_node_int,	                \
+		.export = usbg_set_config_node_int,		        \
```
Comment (mgrzeschik) [inline src/function/uvc.c:109]: These functions should also be implemented as their ..._node_int64 variants.

File: src/function/uvc.c (hunk 1 above)
Comment (any1) [reply src/function/uvc.c:107]: Yeah, I didn't run into this problem because I use `usbg_write_dec64` directly in `uvc_set_frame`.

**Comment by mgrzeschik** [comment]
For compatibility with libconfig I would rather stick to use long long everywhere instead of int64_t.

**Comment by mgrzeschik** [comment]
Actually this is all wrong. It will be nice to have 64 bit support in this lib, but it is unnecessary for uvc. Instead we should make these variables unsigned int. Which is also the type of value we use in the kernel. It would not  be possible to write those 64 bit values into the kernel though.

**Comment by mgrzeschik** [comment]
I repurposed the branch to fix the set_frame callback. The issue with the unsigned format is for another MR.

**Pull request merged**